MySolution.rb

```diff
@@ -0,0 +1,48 @@
+class MySolution
+
+    #problem1
+    def sumOfEven(array)
+        sum = 0
+        for i in array
+            if i.even?
+                sum += i
+            end
+        end
+        return sum
+    end
+    
+    #problem2
+    def getLargest2Numbers(array)
+        bigMax = -1
+        smallMax = -1
+        for i in array
+            if i > bigMax and i > smallMax #if bigger then both
```

Comment:
This comments don't actually bring any benefit. Please write the code in such a way that it reads clearly and it does not require any explanation.

Reply:
Ok, I understood

Reply:
Actually yeah, the comment was redundant